tests/lib/rules/no-useless-escape.js

```diff
@@ -72,8 +72,8 @@ ruleTester.run("no-useless-escape", rule, {
         {code: "var foo = `${foo}\\f`;", parserOptions: {ecmaVersion: 6}},
         {code: "var foo = `${foo}\\\n`;", parserOptions: {ecmaVersion: 6}},
         {code: "var foo = `${foo}\\\r\n`;", parserOptions: {ecmaVersion: 6}},
-        {code: "var foo = `\\\``", parserOptions: {ecmaVersion: 6}},
-        {code: "var foo = `\\\`${foo}\\\``", parserOptions: {ecmaVersion: 6}},
+        {code: "var foo = `\\\``", parserOptions: {ecmaVersion: 6}}, // eslint-disable-line no-useless-escape
+        {code: "var foo = `\\\`${foo}\\\``", parserOptions: {ecmaVersion: 6}}, // eslint-disable-line no-useless-escape
```

Comment:
Similar here, I'll resolve this one to two backslashes. Thanks for all the feedback!